app/main.py

```diff
@@ -1,3 +1,18 @@
+from math import ceil
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+
+    def __init__(self, name: str, description: str, weeks: int):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days: int):
+        return ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict: dict):
+        return cls(course_dict["name"], course_dict["description"],
+                   cls.days_to_weeks(course_dict["days"]))
```

Comment:
It would be better to use `OnlineCourse.days_to_weeks` here. You will discuss this case in the future lectures on OOP